docs-crawler/docs/aggregate_crawler_output.py

```diff
@@ -1,15 +1,28 @@
+# _*_ Coding: utf-8 -*-
+
+# Iterates over the output generated by the Pytorch,
+# TensorFlow and TensorFlow.js crawlers and provided a sorted
+# output of every function.
+
 import json
 import re
 from translations import WORD_TRANSLATIONS, COMMAND_TRANSLATIONS
 
+# Compiler() is the main parent class responsible for
+# all the translation from the respective output files
+# from all the crawlers.
+
 
 class Compiler(object):
-    tf = []
-    tfjs = []
-    torch = []
+    tf = []  # Create empyt array for TensorFlow.
+    tfjs = []  # Create empty array for TensorFlow.js.
+    torch = []  # Create empty array for Pytorch.
```

Comment:
It might be best to leave a single comment that says what is stored in these arrays